src/modm/platform/dma/stm32/dma.hpp.in

```diff
@@ -46,8 +47,11 @@ public:
 	{
 		if constexpr (ID == 1)
 			Rcc::enable<Peripheral::Dma1>();
-		else
+		else {
+			#ifdef DMA2_BASE
```

Comment:
Wait, isn't this always true, since you define this in `dma_hal.hpp` below?

Reply:
Yeah 🤦🏽

I replace the macros with jinja magic anyways...